src/main/java/uk/gov/hmcts/juror/api/moj/service/PrintDataServiceImpl.java

```diff
@@ -229,6 +232,21 @@ public void checkLetterInBulkPrint(String jurorNumber, String formType, LocalDat
         }
     }
 
+    @Override
+    public void removeQueuedLetterForJuror(JurorPool jurorPool) {
+
+        // determine form codes for juror status
+        List<FormCode> formCodes = FormCode.getFormCodesForJurorStatus(jurorPool.getStatus().getStatus());
+
+        List<BulkPrintData> bulkPrintDataList = bulkPrintDataRepository
+            .findByJurorNoAndCreationDateAndFormAttributeFormTypeIn(
+            jurorPool.getJuror().getJurorNumber(), LocalDate.now(),
```

Comment:
Do we need to remove the creation date filter?
And get any letters that have not been printed as letters can take between 2 to 4 days to be sent?

Reply:
The specific use case here is the user has mistakenly deferred/excused someone, they now need to mark as responded and dont want any letters queued today, if they miss this chance, then they are aware the letter will go out and nothing they can do about it